### ISSUE TITLE
Tree

### ISSUE THREAD

**Issue by rexingQ** [koalaman/shellcheck#3357]
[bc1p73d0ve0fg39c9ceaxcmnukdm8670c8dxvzx4f23l62d246j9ahgs8dzs4w_en_usd_2025-06-25_2025-06-25_1-1.pdf](https://github.com/user-attachments/files/23536163/bc1p73d0ve0fg39c9ceaxcmnukdm8670c8dxvzx4f23l62d246j9ahgs8dzs4w_en_usd_2025-06-25_2025-06-25_1-1.pdf)

#### For bugs
- Rule Id (if any, e.g. SC1000): 
- My shellcheck version (`shellcheck --version` or 'online'): online
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### For new checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/your/shebang
Your script
```

#### Here's what shellcheck currently says:
```
TODO: Describe current output
```
#### Here's what I wanted or expected to see:

TODO: Describe expected/desired output

.licenses/npm/@actions/artifact.dep.yml
